src/transformers/training_args_tf.py

```diff
@@ -59,9 +138,15 @@ def _setup_strategy(self) -> Tuple["tf.distribute.Strategy", int]:
     @property
     @tf_required
     def strategy(self) -> "tf.distribute.Strategy":
+        """
+        The strategy used for distributed training.
+        """
         return self._setup_strategy
 
     @property
     @tf_required
     def n_gpu(self) -> int:
+        """
+        The number of GPUs used in this training.
```

Comment:
The number of replicas, not GPUs (means CPU/TPU cores as well). I think we should think to rename that function to `n_replica` or `n_device`. @julien-c WDYT?